tests/Unit/AuditPageTest.php

```
@@ -40,6 +40,55 @@ public function test_register_menu_skips_audit_and_ai_discovery_without_capabili
     }
 
+    public function test_selected_month_accepts_valid_query_value(): void
+    {
+        $_GET['month'] = '2026-08';
+
+        try {
+            $this->assertSame('2026-08', self::callPrivate(AuditPage::class, 'selected_month'));
+        } finally {
+            unset($_GET['month']);
+        }
+    }
```

PHPMD (2.15.0)

[error] test_selected_month_accepts_valid_query_value accesses the super-global variable $_GET. (undefined)

(Superglobals)